src/Search.js

```diff
@@ -14,10 +14,23 @@ const Search = () => {
             <label htmlFor="customerName">Customer name</label>
             <label className="searchIdLabel" htmlFor="customerName">Customer id</label>
           </form>
+          <div className="search-row">
+            <input
+              type="text"
+              id="customerName"
+              className="form-control"
+              placeholder="Customer name"
+            />
+            <button className="btn btn-primary">Search</button>
```

Comment:
This is exactly the same as the `SearchBtn` component - let's re-use that, so that if we change it in the future (e.g. give it a new class), we only need to change it in one place :)